std/container/binaryheap.d

```diff
@@ -201,12 +201,14 @@ Returns $(D true) if the heap is _empty, $(D false) otherwise.
 Returns a duplicate of the heap. The underlying store must also
 support a $(D dup) method.
```

Comment:
Please add a note here that this is only available if the underlying type has it.